src/core_atmosphere/mpas_atm_core.F

```diff
@@ -262,6 +266,15 @@ function atm_core_init(domain, startTimeStamp) result(ierr)
          call mpas_pool_get_array(state, 'xtime', xtime, 1)
          xtime = startTimeStamp
 
+         ! Set the units to be cf compliant 'seconds since <cf-timestamp>'
+         call mpas_pool_get_field(state, 'Time', Time_field) 
+         call mpas_modify_att(Time_field % attLists(1) % attList, 'units', &
+              'seconds since ' // mpas_string_replace(startTimeStamp, '_', ' '))
+
+         ! Initialize Time to 0
+         call mpas_pool_get_array(state, 'Time', Time, 1)
+         Time = 0.
```

Comment:
Since `Time` is read from the restart file, I don't think we want to always reset it to zero here.

Reply:
Following up on this, all real-valued variables declared in the `Registry.xml` file are set to zero after being allocated. But, I think we may want to consider changes to the `init_atmosphere` core (to be included as additional commits in this PR branch) that add a CF-compliant `Time` variable to the ICs, LBCs, and surface update files.